modules/creating_a_blueprint.md

```diff
@@ -65,6 +66,10 @@ A JHipster blueprint:
     - languages
     - spring-controller
     - spring-service
+    - heroku
+    - cypress
+    - page
+    - entities
```

Comment:
I'm sorry I should have been more clear in my last comment.

`entities`, `cypress`, and `page` are only in the beta v7 versions and not v6.  `heroku` is in v6

Reply:
done ;)

Reply:
added app and heroku for the v6 and the other for the v7

Reply:
I think it's correct now, if it is let's merge this pr an will update the v7 and we will be ok with the doc